js/mixins/dom_iterable.ts

```diff
@@ -24,19 +24,19 @@ export function DomIterableMixin<K, V, TBase extends Constructor>(
   // tslint:disable-next-line:variable-name
   const DomIterable = class extends Base {
     *entries(): IterableIterator<[K, V]> {
-      for (const entry of (this as any)[dataSymbol].entries()) {
+      for (const entry of (this as any)[dataSymbol]) {
         yield entry;
       }
     }
 
     *keys(): IterableIterator<K> {
-      for (const key of (this as any)[dataSymbol].keys()) {
+      for (const [key,] of (this as any)[dataSymbol]) {
```

Comment:
#1056 has this as:

```suggestion
      for (const [key] of (this as any)[dataSymbol]) {
```

which is better